### PR TITLE
feat: add default and overscroll backgrounds

### PR DESCRIPTION
**Pull request by gfellerph**

### CODE REVIEW

File: packages/styles/src/elements/body.scss [hunk 1]
```diff
@@ -1,17 +1,28 @@
...
+}
+
 body {
   font-family: tokens.get('body-font-family');
   font-size: tokens.get('body-font-size');
   font-weight: tokens.get('body-font-weight');
   line-height: tokens.get('body-line-height');
   letter-spacing: tokens.get('body-letter-spacing');
   color: tokens.get('body-color');
+  background-color: tokens.get('palettes-color-default-bg', palettes.$post-palettes);
```
Comment (alizedebray) [inline packages/styles/src/elements/body.scss:27]: The background color of the body is already set in the palettes PR: https://github.com/swisspost/design-system/pull/3850/files#diff-8c14f2102052982c3be175ed844b74d595f43df77e4c22707011eecdd5b4ad2dR5

```suggestion
```

Comment (alizedebray) [reply packages/styles/src/elements/body.scss:27]: In the end, I think I had you create this PR for nothing... 🙈 I thought you would need the palettes to fetch the Cargo green for the HTML element.

File: packages/styles/src/elements/body.scss [hunk 2]
```diff
@@ -1,12 +1,22 @@
 @use 'sass:map';
+@use '../variables/color';
 @use '../functions/tokens';
 @use '../tokens/elements';
+@use '../tokens/palettes';
```
Comment (gfellerph) [inline packages/styles/src/elements/body.scss:5]:
```suggestion
```

File: packages/styles/src/elements/body.scss (hunk 1 above)
Comment (gfellerph) [reply packages/styles/src/elements/body.scss]: There's still the background of the `<html>` element that changes, so not entirely for nothing.

**Review by alizedebray — APPROVED** [review]

**Comment by gfellerph** [comment]
@alizedebray retargeting to the palettes PR created a bunch of issues. Maybe better to just go to main?

**Pull request merged**